opentelemetry-sdk/src/logs/logger.rs

```diff
@@ -29,6 +29,9 @@ impl opentelemetry::logs::Logger for SdkLogger {
 
     /// Emit a `LogRecord`.
     fn emit(&self, mut record: Self::LogRecord) {
+        if Context::is_current_telemetry_suppressed() {
```

Comment:
Do we need this check here as well? The appender would anyway call `SdkLogger::event_enabled` before calling `emit` method. So why check this twice?

```suggestion
        #[cfg(not(feature = "spec_unstable_logs_enabled"))]
        if Context::is_current_telemetry_suppressed() {
```

Reply:
Appenders are not obligated to leverage that path. They may check it, they may not check. We can recommend all to do it (and do it in the appenders maintained here), but since we cannot enforce, I think the extra check is okay.

The dual cost is true for the enabled() case - we check it twice, but if Log is enabled, adding 1 ns should be okay?

Reply:
> but since we cannot enforce, I think the extra check is okay.

Yes. Agreed.